telescope/selector_test.py

```diff
@@ -126,8 +127,133 @@ def testValidInput_v1dot1_Simple(self):
+                }
+            },
+            "start_times": ["2014-02-01T00:00:00Z"]
+        }"""
+    selector_expected = selector.Selector()
+    selector_expected.start_time = utils.make_datetime_utc_aware(datetime.datetime(2014, 2, 1))
+    selector_expected.duration = 30 * 24 * 60 * 60
+    selector_expected.metric = 'average_rtt'
+    selector_expected.ip_translation_spec = (
+         iptranslation.IPTranslationStrategySpec('maxmind',
+                                                 {'db_snapshots': ['2014-08-04']}))
+    self.assertParsedSingleSelectorMatches(selector_expected, selector_file_contents)
+
+  def testValidInput_v1dot1_Simple_CountriesCaseInsensitivity(self):
+    selector_file_contents = """{
+            "file_format_version": 1.1,
+            "duration": "30d",
+            "metrics": ["average_rtt"],
+            "ip_translation":{
+                "strategy":"maxmind",
+                "params":{
+                    "db_snapshots":["2014-08-04"]
+                }
+            },
+            "start_times": ["2014-02-01T00:00:00Z"],
+            "client_countries": ["us", "Ca", "uK", "AU"]
+        }"""
+
+    selectors_expected = []
+    selector_base = selector.Selector()
+    selector_base.start_time = utils.make_datetime_utc_aware(datetime.datetime(2014, 2, 1))
+    selector_base.duration = 30 * 24 * 60 * 60
+    selector_base.metric = "average_rtt"
+    selector_base.ip_translation_spec = (
+         iptranslation.IPTranslationStrategySpec('maxmind',
+                                                 {'db_snapshots': ['2014-08-04']}))
+
+    for client_country in ["us", "ca", "uk", "au"]:
+      selector_copy = copy.copy(selector_base)
+      selector_copy.client_country = client_country
+      selectors_expected.append(selector_copy)
+
+    self.assertParsedSelectorsMatch(selectors_expected, selector_file_contents)
+
+  def testValidInput_v1dot1_Simple_SingleLocationValues(self):
+    selector_file_contents = """{
+            "file_format_version": 1.1,
+            "duration": "30d",
+            "metrics": ["average_rtt"],
+            "ip_translation":{
+                "strategy":"maxmind",
+                "params":{
+                    "db_snapshots":["2014-08-04"]
+                }
+            },
+            "sites": ["lga02"],
+            "start_times": ["2014-02-01T00:00:00Z"]
+        }"""
+    selector_expected = selector.Selector()
+    selector_expected.start_time = utils.make_datetime_utc_aware(datetime.datetime(2014, 2, 1))
+    selector_expected.duration = 30 * 24 * 60 * 60
+    selector_expected.metric = 'average_rtt'
+    selector_expected.ip_translation_spec = (
+         iptranslation.IPTranslationStrategySpec('maxmind',
+                                                 {'db_snapshots': ['2014-08-04']}))
+    selector_expected.site = 'lga02'
+    self.assertParsedSingleSelectorMatches(selector_expected, selector_file_contents)
+
+    selector_file_contents = """{
```

Comment:
Can you put these in separate functions? It helps reduce the possibility of tests polluting each other's results.